README.md

```diff
@@ -39,11 +44,27 @@ Further documentation is available on [our docs site](https://docs.ironcorelabs.
 
 Prerequisites:
 
-+ [Rust toolchain installed](https://www.rust-lang.org/tools/install)
-+ `JAVA_HOME` environment variable set
-+ `clang` installed
+- [Rust toolchain](https://www.rust-lang.org/tools/install) installed
+- `JAVA_HOME` environment variable set
+- `clang` installed
+
+From the root of this repository run `cargo build -p ironoxide-java`. The resulting `target/debug/build/ironoxide-java-*/out/java` directory will have the JNI binding code for the Java side and `target/debug` will have the dynamic library file you need to pull into your Java code. It will be named `libironoxide_java.so` or `libironoxide_java.dylib` depending on your environment. This library will only work on the architecture from which it was built.
```

Comment:
Should we recommend building with `--release` here?

Reply:
We have the tests locked to the debug folder, so we'd need to either change that or make sure we're compiling for debug as well. Most people will be downloading from our `releases` page, so I think leaving this as debug might be okay? But I don't feel strongly about it.